modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/addimages/plugin.js

```diff
@@ -234,6 +338,7 @@
 				editor.on('dragover', this._onDragOver.bind(this));
 				editor.on('drop', this._onDragDrop.bind(this));
 				editor.on('paste', this._onPaste.bind(this));
+				editor.on('imageUploaded', this._onImageUploaded.bind(this));
```

Comment:
We don't need to hook onto `imageUploaded` event, I believe the only place from where it is fired is this file. `_onImageUploaded` can be a simple private function, invoked from  `_onUploadComplete`.

Reply:
Ok, I'll look into it

Reply:
So we can do that, but then `_onImageUploaded`  gets called whether you drag and drop and image or you paste content that contains "embed" images: probably not what we want right? (I already pushed that part for I'll undo it if you agree)

Reply:
And that's actually happening because of "my changes" - so I'll see if there's a way to improve that